Rakefile

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env rake
+require 'rubocop'
+desc 'Run tests and rubocop'
+task :default do
+  Rake::Task['rubocop'].invoke
+  Rake::Task['foodcritic'].invoke
+end
+
+task :rubocop do
+  require 'rubocop'
+  puts 'Running RuboCop...'
+  cli = RuboCop::CLI.new
+  cli.run(%w(--fail-level R))
+  puts "\n\n"
+end
+
+task :foodcritic do
+  if Gem::Version.new('1.9.2') <= Gem::Version.new(RUBY_VERSION.dup)
+    sandbox = File.join(File.dirname(__FILE__), %w(tmp foodcritic cookbook))
+    prepare_foodcritic_sandbox(sandbox)
```

Comment:
This is a nice way to ensure other cookbooks don't trip your foodcritic run. :+1: